.gitignore

```diff
@@ -126,3 +126,4 @@ local.properties
 
 # Code Recommenders
 .recommenders/
+/sdk-src/mockserver.log
```

Comment:
what's this, Xander also added a mock server dependency for session timeout tests

Reply:
Don't know!

Reply:
can you move this ignore to another section, it's not under the relevant category, has nothing to do with the Code Recommenders eclipse plugin